packages/teams-js/src/public/video.ts

```diff
@@ -88,11 +88,33 @@ export namespace video {
     notifyError: (errorMessage: string) => void,
   ) => void;
 
+  /**
+   * Predefined failure reasons for preparing the selected video effect
+   * @beta
+   */
+  export enum EffectFailureReason {
+    /**
+     * A wrong effect id is provided
+     */
+    InvalidEffectId = 'InvalidEffectId',
+    /**
+     * The effect can't be initialezed
+     */
+    InitializationFailure = 'InitializationFailure',
+  }
+
+  enum EffectSuccessReason {
+    EffectPrepared = 'EffectPrepared',
+
+    DirectReturned = 'DirectReturned',
```

Comment:
From grammar point of view: `DirectlyReturned` is more accurate.

From semantic point of view: `ReturnedWithoutPromise` or `NoPromiseReturned` is more declarative.

Reply:
updated to "NoPromiseReturned "